xAPI-Data.md

```diff
@@ -313,21 +313,21 @@ The details of each property of a statement are described in the table below.
 <table>
 	<tr><th>Property</th><th>Type</th><th>Description</th><th>Required</th></tr>
 	<tr><td>id</td><td>UUID</td>
-	<td>UUID assigned by LRS if not set by the Activity Provider.</td>
+	<td>UUID assigned by LRS if not set by the Learning Record Provider.</td>
 	<td>Recommended</td></tr>
 	<tr><td><a href="#actor">actor</a></td><td>Object</td>
 	<td>Whom the Statement is about, as an <a href="#agent">Agent</a> or 
-		<a href="#group">Group</a> Object. Represents the "I" in "I Did This".</td>
+		<a href="#group">Group</a> Object.</td>
```

Comment:
I didn't cut it from other explanatory text; I think the table should just be cleaner.  Anyone else have an opinion on this one to sway the tide?